core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -158,10 +158,11 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
           inProgress.put(topicPartition, LogCleaningPaused)
         case Some(state) =>
           state match {
+            case LogCleaningPaused => {}
```

Comment:
@mimaison, can you please check if there is already an issue for the IllegalArgumentException in `truncateTo` and file one if not? Thanks!